conan/tools/gnu/pkgconfigdeps.py

```diff
@@ -397,6 +398,10 @@ def content(self):
             # Filter the build_requires not activated with PkgConfigDeps.build_context_activated
             if require.build and dep.ref.name not in self.build_context_activated:
                 continue
+            if dep.ref.name in self._properties:
+                for prop, value in self._properties[dep.ref.name].items():
+                    dep.cpp_info.set_property(prop, value)
```

Comment:
Super minor nitpick, I just think it reads better, but feel free to ignore me

```suggestion
            for prop, value in self._properties.get(dep.ref.name, {}).items():
                dep.cpp_info.set_property(prop, value)
```